go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	github.com/urfave/cli/v2 v2.25.7
 	github.com/vmihailenco/msgpack/v5 v5.3.5
 	golang.org/x/crypto v0.13.0
+	golang.org/x/exp v0.0.0-20230905200255-921286631fa9
```

Comment:
using the experimental `slices` package for sorting; once we upgrade to Go 1.21 -- this will be part of stdlib and go away.